app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < Devise::SessionsController
+  after_action :log_failed_login, :only => :new
+
+  def create
+    super
+    LoggingHelper.log_event(request,current_user,{event:"user_login"})
```

Comment:
There is a lot of repeated code here. Perhaps we could create a method in application controller like:

    def log_event(event, attrs)
      LoggingHelper.log_event(request, current_user, {event: event}.merge(attrs))
    end

Then you can just do:

    log_event('user_login')
    log_event("user_failed_login", user:request.filtered_parameters["user"])